test/ModuleInterface/ExplicitInterfaceObjC.swiftinterface

```diff
@@ -0,0 +1,13 @@
+// swift-interface-format-version: 1.0
+// swift-module-flags: -module-name ExplicitInterfaceObjC -module-interface-preserve-types-as-written
```

Comment:
Is `-module-interface-preserve-types-as-written` critical to the test case?

Reply:
Nope, will delete, thanks.